hpobench/benchmarks/mo/cnn_benchmark.py

```diff
@@ -0,0 +1,520 @@
+                'n_fc_layers', default_value=3, lower=1, upper=3, log=False
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_0', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_1', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_2', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'batch_size', lower=1, upper=512, default_value=128, log=True
+            ),
+            CS.UniformFloatHyperparameter(
+                'learning_rate_init', lower=10 ** -5, upper=1, default_value=10 ** -3, log=True
+            ),
+            CS.CategoricalHyperparameter(
+                'batch_norm', default_value=False, choices=[False, True]
+            ),
+            CS.CategoricalHyperparameter(
+                'global_avg_pooling', default_value=True, choices=[False, True]
+            ),
+            CS.CategoricalHyperparameter(
+                'kernel_size', default_value=5, choices=[7, 5, 3]
+            )
+
+        ])
+        return cs
+
+    @staticmethod
+    def get_objective_names(self) -> List[str]:
+        return ['accuracy', 'model_size']
+
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+
+        fidelity_space = CS.ConfigurationSpace(seed=seed)
+        fidelity_space.add_hyperparameters([
+            CNNBenchmark._get_fidelity_choices(iter_choice='variable')
+        ])
+        return fidelity_space
+
+    @staticmethod
+    def _get_fidelity_choices(iter_choice: str) -> Tuple[Hyperparameter, Hyperparameter]:
+
+        fidelity1 = dict(
+            fixed=CS.Constant('budget', value=25),
+            variable=CS.UniformIntegerHyperparameter(
+                'budget', lower=1, upper=25, default_value=25, log=False
+            )
+        )
+
+        budget = fidelity1[iter_choice]
+        return budget
+
+    @staticmethod
+    def get_meta_information() -> Dict:
+        """ Returns the meta information for the benchmark """
+        return {
+            'name': 'Bag of baselines for multi-objective joint neural architecture search and hyperparameter optimization',
+            'references': ['@article{guerrero2021bag,'
+                           'title   = {Bag of baselines for multi - objective joint neural architecture search and hyperparameter optimization},'
+                           'author  = {Guerrero-Viu, Julia and Hauns, Sven and Izquierdo, Sergio and Miotto, Guilherme and Schrodi, Simon and Biedenkapp, Andre and Elsken, Thomas and Deng, Difan and Lindauer, Marius and Hutter, Frank},},'
+                           'journal = {arXiv preprint arXiv:2105.01015},'
+                           'year    = {2021}}',
+                           ],
```

Comment:
Could you please add line breaks, so that the 120 chars per line limit is fulfilled.

Reply:
in general, also please check the code with;

``` bash
flake8 --max-line-length=120 ./hpobench
pycodestyle --max-line-length=120 ./hpobench
pylint --disable=all --enable=W0221 ./hpobench
```